### PR TITLE
Add feature: Print breakpoint table

### PR DESCRIPTION
**Pull request by yihuajack**
Take example5.evcd, breakpoints=[9,11,17] as example, the table is:

```
time    9       11      17
scope   testbench
scope   pe8
req_I   128     16      4
req_O   zzzzzzzz        zzzzzzzz        zzzzzzzz
en_I    1       1       1
en_O    z       z       z
scope   pe4
req_I   8       4       2
req_O   zzzz    zzzz    zzzz
en_I    1       1       1
en_O    z       z       z
```

### CODE REVIEW

**Comment by yihuajack** [comment]
The column width of tables can be fine-adjusted, but to achieve it, sootty needs to recursively see all the wires twice, so performance is lost.

File: sootty/storage/wiregroup.py
Comment (Ben1152000) [inline sootty/storage/wiregroup.py:39]: This change to how wire names are computed should be a new PR, assuming it's separate from the breakpoint logic.

File: sootty/storage/wiregroup.py
Comment (Ben1152000) [inline sootty/storage/wiregroup.py:55]: Add docstring comment for what the function does.

**Review by Ben1152000 — CHANGES_REQUESTED** [review]

**Comment by Ben1152000** [comment]
Have you tested if this works when the waveform is being displayed? The svg is written to stdout and piped to the display program, so if the breakpoints are also printed to stdout that could possibly break the image.

File: sootty/storage/wiregroup.py
Comment (yihuajack) [reply sootty/storage/wiregroup.py:39]: now resolved in: https://github.com/Ben1152000/sootty/pull/64

**Comment by yihuajack** [comment]
> Have you tested if this works when the waveform is being displayed? The svg is written to stdout and piped to the display program, so if the breakpoints are also printed to stdout that could possibly break the image.

Yes, I've tested that this works with or without `-o`, but I'm not quite familiar with `subprocess`. What if `subprocess.run` is used instead of `subprocess.Popen()`?

**Review by Ben1152000 — APPROVED** [review]

File: sootty/storage/wiregroup.py
Comment (Ben1152000) [inline sootty/storage/wiregroup.py:40]: Can you remove the duplicate changes from PR #64? I approved so once you do that you can merge it in.

**Review by Ben1152000 — APPROVED** [review]

File: sootty/storage/wiregroup.py
Comment (yihuajack) [reply sootty/storage/wiregroup.py:40]: Removed

**Pull request merged**
